packages/engine-test-utils/src/__tests__/pods-core/GithubPod.spec.ts

```diff
@@ -0,0 +1,185 @@
+import { ENGINE_HOOKS } from "../../presets";
+import { runEngineTestV5 } from "../../engine";
+import { GithubImportPod } from "@dendronhq/pods-core";
+import { NoteUtils, VaultUtils } from "@dendronhq/common-all";
+
+describe("GithubPod import pod", () => {
+  let result: any;
+  let fname: string;
+  beforeEach(() => {
+    result = {
+      search: {
+        pageInfo: {
+          hasNextPage: false,
+        },
+        edges: [
+          {
+            node: {
+              title: "Test Issue",
+              url: "https://github.com/dendronhq/dendron/issues/902",
+              number: 902,
+              state: "OPEN",
+              labels: {
+                edges: [
+                  {
+                    node: {
+                      name: "area.misc",
+                    },
+                  },
+                ],
+              },
+            },
+          },
+        ],
+      },
+    };
+    fname = "github.issues.902-Test Issue";
+  });
+
+  test("Bad Credentials", async () => {
```

Comment:
Does this test make an actual API call to Github? If so, it's better to tag this test as test.skip in the main test pass, or else it may cause some test reliability issues during the test pass.

Reply:
Yes it would try to make an API call to github. I'll add test.skip to avoid reliability issues. Thanks for catching this

Reply:
@kevinslin : please find the reason to skip the testcase in above comments.

Reply:
@kevinslin Please find the reason to skip the testcase in above comments.